.github/workflows/test.yml

```
@@ -699,6 +698,17 @@ jobs:
           flags: client,${{ matrix.os }},${{ matrix.queryEngine }}
           name: client-${{ matrix.os }}-${{ matrix.queryEngine }}
 
+      - name: Test packages/client/functional
+        if: ${{ contains(needs.detect_jobs_to_run.outputs.jobs, '-all-') || contains(needs.detect_jobs_to_run.outputs.jobs, '-client-') }}
+        run: pnpm run test:functional -- --jest-args='--testTimeout=40000'
```

I think it should be
```suggestion
        run: pnpm run test:functional --jest-args='--testTimeout=40000'
```
with pnpm 7